tests/unit/modules/test_mean_absolute_error.py

```diff
@@ -0,0 +1,91 @@
+        self.mae_calculator.set_params(offset=24, rolling=True, window=2)
+        self.assertEqual(self.mae_calculator.get_params(),
+                         {'offset': 24, 'rolling': True, 'window': 2})
+
+    def test_transform_rolling(self):
+        self.mae_calculator.set_params(rolling=True, window=2)
+        time = pd.to_datetime(['2015-06-03 00:00:00', '2015-06-03 01:00:00',
+                               '2015-06-03 02:00:00', '2015-06-03 03:00:00',
+                               '2015-06-03 04:00:00'])
+
+        test_data = xr.Dataset({"testCol": ("time", xr.DataArray([-2, -1, 0, 1, 2])),
+                                "predictCol1": ("time", xr.DataArray([2, -3, 3, 1, -2])),
+                                "predictCol2": ("time", xr.DataArray([4, 4, 3, -2, 1])), "time": time})
+
+        test_result = self.mae_calculator.transform(y=test_data['testCol'], gt=test_data['testCol'],
+                                                    pred1=test_data['predictCol1'],
+                                                    pred2=test_data['predictCol2'])
+        expected_result = xr.DataArray(np.array([[np.nan, np.nan, np.nan],
+                                                 [0.0, 3, 5.5],
+                                                 [0.0, 2.5, 4],
+                                                 [0.0, 1.5, 3],
+                                                 [0.0, 2, 2]]),
+                                       coords={"time": time, "predictions": ["gt", "pred1", "pred2"]},
+                                       dims=["time", "predictions"])
+
+        xr.testing.assert_allclose(test_result, expected_result)
+
+    def test_transform(self):
+        self.mae_calculator.set_params()
+
+        time = pd.to_datetime(['2015-06-03 00:00:00', '2015-06-03 01:00:00',
+                               '2015-06-03 02:00:00', '2015-06-03 03:00:00',
+                               '2015-06-03 04:00:00'])
+
+        result_time = pd.to_datetime(['2015-06-03 04:00:00'])
+
+        test_data = xr.Dataset({"testCol": ("time", xr.DataArray([-2, -1, 0, 1, 2])),
+                                "predictCol1": ("time", xr.DataArray([2, -3, 3, 1, -2])),
+                                "predictCol2": ("time", xr.DataArray([4, 4, 3, -2, 1])), "time": time})
+
+        test_result = self.mae_calculator.transform(y=test_data['testCol'], gt=test_data['testCol'],
+                                                    pred1=test_data['predictCol1'],
+                                                    pred2=test_data['predictCol2'])
+
+        expected_result = xr.DataArray(np.array([[0.0, 2.6, 3.6]]),
+                                       coords={"time": result_time, "predictions": ["gt", "pred1", "pred2"]},
+                                       dims=["time", "predictions"])
+
+        xr.testing.assert_equal(test_result, expected_result)
+
+    def test_transform_without_predictions(self):
+        self.mae_calculator.set_params()
```

Comment:
This lines is not unnecessary

Reply:
Now, changelog and docs should be updated (have lines 73 to 76 been missing and is there still something missing?)